README.md

```diff
@@ -147,6 +148,16 @@ To resolve service worker types, just add `WebWorker` to lib entry on your `tsco
 "lib": ["ESNext", "DOM", "WebWorker"],
 ```
 
+### Static assets handling
+
+By default, all icons on `PWA Manifest` option found under Vite's `publicDir` option directory, will be included 
+in the service worker *precache*. You can disable this option using `includeManifestIcons: false`.
+
+You can also add another static assets such as `favicon`, `svg` and `fonts` files using `include` option.
+You don't need to configure `PWA Manifest icons` on `include` option.
```

Comment:
Add entry specifiying that static assets must be under Vite's `publicDir` option directory.